director/urls.py

```diff
@@ -53,6 +61,10 @@
     # Home page
     path('', HomeView.as_view(), name='home'),
 
+    # Privacy policy
+    path('privacy-policy/', PrivacyView.as_view(), name='privacy-policy'),
```

Comment:
To avoid too may top level paths, I suggest nesting these under an `about` top level url path e.g `about/privacy`